ggshield/cmd/iac/scan/all.py

```diff
@@ -85,7 +85,7 @@ def iac_scan_all(
     client = ctx.obj["client"]
 
     scan_parameters = IaCScanParameters(
-        list(config.user_config.iac.ignored_policies),
+        list({ignored.policy for ignored in config.user_config.iac.ignored_policies}),
```

Comment:
```suggestion
        [ignored.policy for ignored in config.user_config.iac.ignored_policies],
```
Unless it is possible to have duplicate value in  user_config.iac.ignored_policies.

Reply:
Skipped as we still accept duplicate values